### PR TITLE
IAA Upgrade

### PR DESCRIPTION
**Pull request by Minijar**
Adds the ability for IAA to represent a company of their choosing, similar to how a chaplain chooses a Deity. Also adds alt titles for IAA: Sol Gov Rep, Health & Safety Inspector and Company Representative.

Due to a buggering up on my behalf, also includes my medical post pr, shouldn't be a problem once that gets merged (If).

Cheers to Ragnie for the idea.

### CODE REVIEW

**Comment by Finalsong** [comment]
:thumbs_up: :100:

**Comment by Helidor** [comment]
Is it too late to suggest an HR representative? Less clinical than IA, and more so advocating for the crew's interests? I've been dying to have that be an option and the only current role similar are IA and HoP, each with their own drawbacks.

**Comment by Mark9013100** [comment]
Crew<s>Hu</s>man Resources!

**Comment by Kelenius** [comment]
*cough* HoP *cough*

**Comment by zaredman** [comment]
I'd keep HR Rep out exclusively due to the fact that the purpose behind HR reps is pretty much what a HOP is *supposed* to do.

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -1605,6 +1605,10 @@ area/space/atmosalert()
 	icon_state = "engineering_aft_hallway"
 
 
+/area/engineering/monitoring_station
```
Comment (Zuhayr) [inline code/game/area/Space Station 13 areas.dm]: What are these areas?

**Comment by Zuhayr** [comment]
DNM pending #977.

File: code/game/jobs/job/civilian.dm [hunk 2]
```diff
@@ -246,7 +246,7 @@
 	economic_modifier = 7
 	access = list(access_lawyer, access_sec_doors, access_maint_tunnels, access_heads)
 	minimal_access = list(access_lawyer, access_sec_doors, access_heads)
-
+	alt_titles = list("Sol Gov Representative","Health & Safety Inspector","Company Representative")
```
Comment (Zuhayr) [inline code/game/jobs/job/civilian.dm]: Please specifically test Health & Safety Inspector, I am unsure if & will fuck up that code or not.

File: code/game/jobs/job/civilian.dm [hunk 3]
```diff
@@ -255,14 +255,28 @@
...
+		if (H.mind.role_alt_title)
+			switch(H.mind.role_alt_title)
+				if("Company Representative")
+					if(H.gender == FEMALE)
+						H.equip_to_slot_or_del(new /obj/item/clothing/under/suit_jacket/female(H), slot_w_uniform)
+					else
+						H.equip_to_slot_or_del(new /obj/item/clothing/under/suit_jacket/really_black(H), slot_w_uniform)
+					spawn(0)
+						var/company = "Nanotrasen"
+						var/new_company = sanitize(input(H, "As a company representative, what company do you represent? The default is Nanotrasen", "Name change", company), MAX_NAME_LEN)
```
Comment (Zuhayr) [inline code/game/jobs/job/civilian.dm]: Needs a full stop after NanoTrasen and a capital T.

**Comment by Zuhayr** [comment]
Other than little notes, conflicts and included commits, looks fine. :+1:

**Comment by Minijar** [comment]
Going to have to refork this and reopen and combine it with #977  , whenever something else mapping wise is merged before this, it gets conflicts.

**Pull request closed without merging**